contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -243,11 +262,14 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
             configuration.incentive,
             conversionConfig.incentive,
             configuration.enabled,
-            conversionConfig.enabled
+            conversionConfig.enabled,
+            configuration.onlyForPrivateConversions,
+            conversionConfig.onlyForPrivateConversions
         );
 
```

Comment:
`onlyForPrivateConversions`: `true` - Conversions are open for private conversions not for users.
`onlyForPrivateConversions`: `false` - Conversions are open for both private conversions as weel as for users.

In this way `PrivateConversions` are always valid, so we need to check for `converterNetwork` address whenever new conversion config is added rather than when `onlyForPrivateConversions` is true.

Reply:
I have added a check in the `setConversionConfig` :
```suggestion
if (address(converterNetwork) == address(0)) {
            revert InvalidConverterNetwork();
        }
```